public/app/features/dashboard/panel_editor/QueriesTab.tsx

```diff
@@ -211,6 +212,10 @@ export class QueriesTab extends PureComponent<Props, State> {
     this.forceUpdate();
   };
 
+  onTransformersChange = (transformers: Array<DataTransformerConfig<any>>) => {
+    this.props.panel.setTransformations(transformers);
```

Comment:
They shouldn't I think. But we also need to have to update the PanelModel's transformations so that it's saved later on - this approach with passing more complex operations via props is sth that we usually do and is also easier to test (if we  were to test this component) as you don't have to mock entire PanelModel.